tests/test_cameras.py

```diff
@@ -117,6 +118,34 @@ def setUp(self) -> None:
         torch.manual_seed(42)
         np.random.seed(42)
 
+    def test_look_at_view_transform_from_eye_point_tuple(self):
+        dist = math.sqrt(2)
+        elev = math.pi / 4
+        azim = 0.0
+        eye = ((0.0, 1.0, -1.0), )
```

Comment:
Now that the `camera_position_from_spherical_angles` function has been modified so that z is positive, you will need to change eye to `eye = ((0.0, 1.0, 1.0), )` in order for the tests to pass.

Reply:
hi @nikhilaravi, I've made that required change.